lib/node-onvif.js

```diff
@@ -167,7 +167,17 @@ Onvif.prototype._execCallback = function(callback, arg1, arg2) {
 	}
 };
 
-Onvif.prototype._sendProbe = function(callback) {
+Onvif.prototype._sendProbe = function(address, callback) {
+	if (typeof address === 'function') {
+		callback = address;
+		address = undefined;
+	}
+	if (!address) {
+		address = this._MULTICAST_ADDRESS;
+Onvif.prototype._sendProbe = function(callback, address) {
+	if (!address) {
+		address = this._MULTICAST_ADDRESS;
+	}
```

Comment:
Looks like this got a little messed up, I think you should be good to delete 170-176